x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/register.ts

```diff
@@ -61,6 +65,13 @@ export function registerCspmUsageCollector(
         getAlertsStats(collectorFetchContext.esClient, logger),
       ]);
 
+      const indicesStats = handleResult('Indices', results[0]);
+      const accountsStats = handleResult('Accounts', results[1]);
```

Comment:
here if there's an error you would do something like:

```ts
const accountsStats = handleResult('Accounts', results[1]) || []
```

Reply:
`Promise.allSettled()` always resolves in a truthy value - an object so I would never render a  `[]`.
```
 // [
//   { status: 'fulfilled', value: 33 },
//   { status: 'fulfilled', value: 66 },
//   { status: 'fulfilled', value: 99 },
//   { status: 'rejected', reason: Error: an error }
// ] ```